precompiles/utils-macro/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "precompile-utils-macro"
```

Comment:
I almost think it could go in the existing precompile utils crate. I know we discussed maybe moving that stuff to Frontier or ORML but they would likely go there together.

Reply:
Procedural macros must necessarily be in a special proc-macro crate, so it is impossible to put them in the existing precompile-utils crate.
On the other hand, I can make a sub-crate of it and re-expose it in precompile-utils crate

Reply:
I see, I didn't know that. I don't want to make you waste time moving your code around left and right. I think it's good where it is.

Reply:
I moved the crate as a sub-crate of precompile-utils and I re-export the macro in precompile-utils, this allows other crates (like rewards precompile), to use the macro through precompile-utils :)